lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -375,7 +376,13 @@ public void run(LottieComposition composition) {
       });
       return;
     }
-    animator.playAnimation();
+
+    if (systemAnimationsEnabled || getRepeatCount() == 0) {
```

Comment:
Actually I don't understand why we play the animation when it doesn't repeat.  Shouldn't we disable playing all the time when system animations are disabled?

Reply:
With this solution, if the dev added events at starting, ending or at several point in the animation, they will be called. 

If I don't play the animation at all, no event will be called (we had this problem in our tests). 

There is still issues if there is events in a repeated animation but I didn't handled this case as I think it's overkill and it should not happen really frequently